loops-and-arrays.js

```diff
@@ -4,6 +4,12 @@
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
 
+const names = ["Henk", "Piet", "Fred", "Joop", "Ad", "Peet", "Pieter", "Klaas"];
```

Comment:
Bij alle 3 de opdrachten was het de bedoeling dat de console.log buiten de for-loop  geplaatst moest worden en aangeroepen met dezelfde naam als de variabel. Nu staat je console.log bij alle 3 de opdrachten in je loop en heb je in de log de aanpassing gemaakt. Als je bij het eerste voorbeeld buiten je for console.log(names) zou doen krijg je ["Henk", "Piet", "Fred", "Joop", "Ad", "Peet", "Pieter", "Klaas"], zonder je erachter.